### ISSUE TITLE
Automatically (re-)generate POT files

### ISSUE THREAD

**Issue by westonruter** [xwp/wp-dev-lib#46]

**Comment by fjarrett** [comment]
:+1: for this feature! It will be so handy to do this on the `pre-commit` hook.

**Comment by westonruter** [comment]
I don't think this is relevant anymore with the new reality of translate.wordpress.org

**Comment by valendesigns** [comment]
I would do it with Grunt anyhow. This might be relevant for private repos that need transitions though.

**Comment by fjarrett** [comment]
It's very relevant for private plugins, as well as free plugins alike that already have PO files and elect not to crowdsource the translations.

**Comment by westonruter** [comment]
OK :smile:
